_includes/head.html

```diff
@@ -16,4 +16,22 @@
   <link rel="manifest" href="{{ site.url }}/manifest.json">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
   <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
-</head>
+  {% if page.layout == 'permalink' %}
+  <meta name="twitter:card" content="summary">
+  {% else %}
+  <meta name="twitter:card" content="summary_large_image">
+  {% endif %}
+  <meta name="twitter:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}">
```

Comment:
Are you referring to the `{% if page.title  %}` on `line 18`? I think that was the way it was originally so I left it as is. But I see why it makes sense to wrap the `<title>` tag with an `{% if page.title %}` conditional.